lib/handlers/buildModels.js

```diff
@@ -184,30 +184,6 @@ function buildModelsWrapper(modelBucket, modelFetcher) {
                             failedModels.push(op.result);
                         }
                     });
-
```

Comment:
It's been awhile since I looked at conductor code - IIRC, are the failedModels directly accessible from userland? My only concern would be whether or not this ends up swallowing real issues, but if you have access to the failedModels then you should be able to tactically address one offs if needed.

Reply:
yes, they are--we have logging in the app that logs essentially the same thing you see here, but properly scrubbed

Reply:
👍